permissions/plugin-smalltest.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "smalltest"
+paths:
+- "com/smalltest/smalltest"
```

Comment:
Not a blocker, but I can't help but wonder about the wisdom of this when the domain smalltest.com is for sale. 

https://maven.apache.org/guides/mini/guide-naming-conventions.html

Reply:
@daniel-beck Thank you. We are aware of the domain issue. Can we get approved without having to purchase that domain?

Reply:
@SmallTestHQ Done.